AnagramSolver.Settings/AnagramSolver.Settings.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You have created a project in the solution. Then you removed from the solution. However, the files are still here. These files are just not in "Solution" so that why you can't see in Solution Explorer